tock/hours/views.py

```diff
@@ -800,26 +802,30 @@ def last_timecard(self):
 
     def prefilled_formset(self):
         timecard = self.last_timecard()
-        project_ids, extra = [], 1
+        prefills = dict(TimecardPrefillData.objects.filter(
```

Comment:
I honestly don't know, but would using values() rather than values_list save you some further conversion to a dict here?

Reply:
Ha, it's funny you bring this up because @annalee and I went back and forth on this very piece!  I'll take another quick look just to see, but for the moment this at least works. :-)

Reply:
Ah I know why we did this - we needed the project ID itself to serve as the key in this dictionary, not the field name itself that we were pulling.